project/src/main/ui/settings-language.gd

```diff
@@ -11,7 +13,40 @@ func _ready() -> void:
 		_option_button.add_item(TranslationServer.get_locale_name(locale))
 	
 	# update the option button's selection to the current locale
-	_option_button.selected = TranslationServer.get_loaded_locales().find(TranslationServer.get_locale())
+	_option_button.selected = TranslationServer.get_loaded_locales().find(_current_loaded_locale())
```

Comment:
It might be worth spitting out an error if it doesn't exist. We don't want this to bite us in the future.

Reply:
Added error, `Locale '%s' was not in the list of loaded locales`